### DISCUSSION TITLE
citation for SSMSE

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [nmfs-ost/SSMSE#84] [Ideas]
After the last SSMSE meeting I have been thinking quite a bit about citations and attempting to find the best way forward for R packages. I came across [this blog post](https://ropensci.org/blog/2021/02/16/package-citation/) (thanks to Kathryn) about how to get people to cite a package. The main thing seems to be to make it clear to users that there is a citation. But, they do list a few journals that are meant to enable citations of software. Where you push a publication to depends on your goals imo. Do you want people to be made aware of SSMSE or do you want recognition via citations. I think if you want the former, then you should push for a publication in a fisheries journal rather than a software journal.

**Comment by k-doering-NOAA** [comment]
Good thoughts as always on this. I think we could also do both, publishing first in a software journal (because this won't require as much prep), then in a fisheries journal at a later date (with more material besides the code). 

I think my main concern (Nathan, feel free to contradict me) is folks not knowing how they can cite SSMSE when they use it. I think trying to make more folks aware of the tool can come later, perhaps?.

**Reply by kellijohnson-NOAA** [reply]
We were pretty lucky I think with ss3sim that Sean pushed to publish in plosone, so we kind of killed two birds with one stone there.

**Comment by kellijohnson-NOAA** [comment]
I think that r4ss, ss3sim, and SSMSE could all use the citation file as a way to increase citations because there is no information when you pull down the drop down menu on the home page for each package. FYI this drop down menu is located in the upper right "About" section. Guidance is available [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) on how to use this feature.

**Comment by kellijohnson-NOAA** [comment]
Also, big picture. I think that there could be a third publication if you are interested in summarizing all of the MSE frameworks that are available. #loftygoals 🥇

**Reply by k-doering-NOAA** [reply]
That would be a helpful review paper!
